lib/fetching/content-services/facebook-content-service.js

```diff
@@ -1,39 +1,46 @@
-   API Reference. We use this field to reduce the number of requests needed to fetch both posts and comments.
+This content service uses the Facebook Graph API to grab data from both Facebook pages and groups.
+There are many usage limitations set by the API which this file must acknowledge if any changes
+are to be made, the details are found below:
+
+  - The way in which the API handles requests for Groups and Pages differ. The general idea is
+    that Pages are static and Groups are dynamic. More specifically, the "since" parameter used
+    in the GET request has different meanings for the two different objects. For Groups, the API grabs
+    all posts where updated_time >= "since value". For Pages, the API grabs all posts where
+    created_time >= "since value".
+
+  - There are usage limitations set by the API that limit the amount of requests that the app can make
+    per hour, if the limit is reached, calls will be blocked for 30 mins. In general, send out as few
+    requests as possible and keep them light.
+  Details on rate limitation can be found here:
+  https://developers.facebook.com/docs/marketing-api/api-rate-limiting
+
+  - The "comments" GET field parameter being used in the current implementation is not documented
+    in the Graph API Reference. We use this field to reduce the number of requests needed to fetch
+    both posts and comments.
 
   - Posts are by default limited to 10 per request, the rest can be retrieved by using the paging field.
 
-The current implementation is as follows: The service fetches one page worth of most recent posts(default: 10), then waits
-on the interval to fetch another 10. Any duplicate posts will be filtered out by the _lastReportDate.
+The current implementation is as follows: The service fetches one page worth of most recent posts
+(default: 10), then waits on the interval to fetch another 10. Any duplicate posts will be filtered
+out by the _lastReportDate.
 
   - Service does not use the "since" param because of the different responses for the different object types.
 
   - Service does not use paging because of usage limits.
 
-  - This service will miss any posts beyond 10 that have been made since the last fetch. e.g If 12 posts were made since
-  the last fetch the 1rst and 2nd(oldest) posts will be missed.
+  - This service will miss any posts beyond 10 that have been made since the last fetch.
+  e.g If 12 posts were made since the last fetch the 1rst and 2nd(oldest) posts will be missed.
 
-  - This service will miss comments on any posts more than 10 posts old. Comments are attached the posts, so no comments
-  on any posts older than the 10th oldest one will be retrieved.
+  - This service will miss comments on any posts more than 10 posts old.
+    Comments are attached the posts, so no comments on any posts older than the 10th oldest one
+    will be retrieved.
 
   - It will also miss any comments beyond 10 that have been made since the last fetch.
 */
+'use strict';
```

Comment:
I think if we're going to strict mode here we need to fix the use of `this` that eslint notices.